openhands/storage/docker_snapshots.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python3
```

Comment:
this probably belongs in `openhands/runtime/impl/docker/`. Storage is more for application-level persistent data